### ISSUE TITLE
WF Lookup & Score: Gene - related_to - SmallMolecule (CHEBI:45783)

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/testing#117]
**Query:** [workflow3.json](https://github.com/NCATSTranslator/testing/blob/main/ars-requests/not-none/1.1/workflow3.json)
**PK:** b1bb2176-5da3-46d0-b61c-64ca87604500
**Control:**  
[Results Tracking Sheet](https://docs.google.com/spreadsheets/d/1Y2yfUFl2SIgv5KkyxdQB94UE3HtVdJNJ7KdnwQgZcYI/edit?usp=sharing)

![image](https://user-images.githubusercontent.com/38321826/133148872-8d54b9de-40d6-482d-90ac-447854b6061a.png)

**Comment by sstemann** [comment]
@webyrd including Unsecret Agent here, since the results are un-scored

**Comment by andrewsu** [comment]
Because BTE does not implement an independent "score" workflow operation, it returns a 400 error as designed.  (My understanding from last week's standup is that we were going to consider creating a new workflow operation that was essentially the combination of "lookup" and "score", and _that_ operation would be the default baseline for every ARA...)

**Comment by dkoslicki** [comment]
>My understanding from last week's standup is that we were going to consider creating a new workflow operation that was essentially the combination of "lookup" and "score", and _that_ operation would be the default baseline for every ARA...

Correct: we will be addressing the lookup+score issue today during the working group meeting.
